content/scalers/mysql.md

```diff
@@ -0,0 +1,101 @@
++++
+fragment = "content"
+weight = 100
+title = "MySQL"
+background = "light"
++++
+
+Scale applications based on MySQL query result.
+
+**Availability:** v1.1+ | **Maintainer:** Community
+
+<!--more-->
+
+### Trigger Specification
+
+This specification describes the `mysql` trigger that scales based on result of MySQL query.
+The trigger requires the following information:
+- `query` - a MySQL query that should return single numeric value that will be used for HPA
+- `queryValue` - a threshold that is used as `targetAverageValue` in HPA. 
+- Connection information as `dsnConnectionStr` or set of required params (more information below)
+
+#### Using `dsnConnectionStr`
+One option to connect to MySQL database is to use dsn connection string from environment variable:
+```yaml
+triggers:
+  - type: mysql
+    metadata:
+      dsnConnectionStr: "SQL_CONN_STR" // SQL_CONN_STR env variable should point to valid connection string
+      queryValue: "4"
+      query: "SELECT CEIL(COUNT(*) / 6) FROM task_instance WHERE state='running' OR state='queued'"
+```
+
+#### Using authentication parameters
+Other possibility is to pass all required parameters to build a new connection string:
+```yaml
+triggers:
+  - type: mysql
+    metadata:
+      username: "root"
+      password: "MYSQL_PASSWORD" // MYSQL_PASSWORD env variable should point to the password
+      host: "mysql"
+      port: "3306"
+      dbname: "stats_db"
+      queryValue: "4"
+      query: "SELECT CEIL(COUNT(*) / 6) FROM task_instance WHERE state='running' OR state='queued'"
+```
+In this case the following additional parameters must be passed:
+- `username` used to access MySQL database
```

Comment:
All of these should be documented here: https://github.com/kedacore/keda-docs/pull/72/files#diff-0015a92fc8bd54f11e712f524fd72fc9R20

Reply:
We tend to list all of the optional fields in one list so that folks know what to use when